bitrise.yml

```diff
@@ -249,7 +249,19 @@ workflows:
         - is_clean_build: 'no'
         - generate_code_coverage_files: 'yes'
         - project_path: DatadogSessionReplay/Package.swift
-        - xcpretty_test_options: --color --report html --output "${BITRISE_DEPLOY_DIR}/DatadogSessionReplay-tests.html"
+        - xcpretty_test_options: --color --report html --output "${BITRISE_DEPLOY_DIR}/DatadogSessionReplay-unit-tests.html"
+    # TODO: RUMM-3061 Enable snapshot tests after implementing solution for storing large files
+    # - xcode-test:
+    #     title: Run snapshot tests for Session Replay - iOS Simulator
+    #     run_if: '{{enveq "DD_RUN_SR_UNIT_TESTS" "1"}}'
+    #     inputs:
+    #     - scheme: SRSnapshotTests
+    #     - destination: platform=iOS Simulator,name=iPhone 14,OS=16.2
+    #     - should_build_before_test: 'no'
+    #     - is_clean_build: 'no'
+    #     - generate_code_coverage_files: 'yes'
+    #     - project_path: DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests.xcworkspace
+    #     - xcpretty_test_options: --color --report html --output "${BITRISE_DEPLOY_DIR}/DatadogSessionReplay-snapshot-tests.html"
```

Comment:
As discussed on Slack, we will keep it disabled until we solve the problem of storing large files in a sustainable way. A separate task was created for this `RUMM-3061`.